recipes-extend/host-readme/files/Readme.md

```diff
@@ -0,0 +1,32 @@
+# RZ/G2L-SBC host folder
+
+## Description
+
+This directory contains files and tools used for managing and deploying images to the RZ/G2L-SBC platform. It includes build artifacts, manifests, test data, and documentation.
+
+## A top-level directory of host
+
+```
+host
+├── build
+│   ├── ${IMAGE_BASENAME}-rzg2l-sbc-20240717204209.rootfs.manifest                                       # Manifest file for the root filesystem
+│   ├── ${IMAGE_BASENAME}-rzg2l-sbc-20240717204209.testdata.json                                         # Test data for the image
+│   ├── ${IMAGE_BASENAME}-rzg2l-sbc.manifest -> ${IMAGE_BASENAME}-rzg2l-sbc-20240717204209.rootfs.manifest        # Symlink to the root filesystem manifest
+│   └── ${IMAGE_BASENAME}-rzg2l-sbc.testdata.json -> ${IMAGE_BASENAME}-rzg2l-sbc-20240717204209.testdata.json     # Symlink to the test data JSON
+├── Readme.md                                                                                   # This document
+├── env                             
+│   ├── ${IMAGE_BASENAME}.env                                                                       # Environment file specific to ${IMAGE_BASENAME} in yocto build
+│   └── Readme.md
+├── src                                                                                         # Build script folder
+│   ├── git_patch.json
+│   ├── jq-linux-amd64
+│   ├── patches
+│   ├── README.md
+│   ├── rzsbc_yocto.sh
+│   └── site.conf
+└── tools/                                                                                      # Tools and scripts used for managing and flashing bootloaders and filesystems across different platforms
+```
+
+## Note
+
```

Comment:
Can leave the README as default but need to add a line here to explain more about the hierarchy, like this is just a `core-image-*` target and when building another target, a new file will be placed and the same location.